ext/CEGUI/IrrlichtBaW/IrrlichtBaWTexture.cpp

```diff
@@ -0,0 +1,50 @@
+#include "IrrlichtBaWTexture.hpp"
+
+using namespace irr;
+using namespace ext;
+using namespace cegui;
+
+const CEGUI::String& IrrlichtBaWTexture::getName() const
+{
+
+}
+
+const CEGUI::Sizef& IrrlichtBaWTexture::getSize() const
+{
+
+}
+
+const CEGUI::Sizef& IrrlichtBaWTexture::getOriginalDataSize() const
+{
+
+}
+
+const CEGUI::Vector2f& IrrlichtBaWTexture::getTexelScaling() const
+{
+
+}
+
+void IrrlichtBaWTexture::loadFromFile(const CEGUI::String& filename, const CEGUI::String& resourceGroup)
+{
+
+}
+
+void IrrlichtBaWTexture::loadFromMemory(const void* buffer, const CEGUI::Sizef& buffer_size, PixelFormat pixel_format)
+{
+
+}
+
+void IrrlichtBaWTexture::blitFromMemory(const void* sourceData, const CEGUI::Rectf& area)
+{
+
+}
+
+void IrrlichtBaWTexture::blitToMemory(void* targetData)
+{
+
+}
+
+bool IrrlichtBaWTexture::isPixelFormatSupported(const PixelFormat fmt) const
+{
+
+}
```

Comment:
these should be ridiculously simple for you to implement ;)

Start here.

Reply:
resolved